object-browser-schema.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  additionalProperties: false,
```

Comment:
When it is false then you are not allowed to add extra properties to the object. If you do your schema is invalid - the sad dinosaur error in bunsen. The big advantage is that typos get caught earlier and easier.

If there are other things to add we can always extend the schema.